releng/release-eclipse.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -u # run with unset flag error so that missing parameters cause build failure
+set -e # error out on any failed commands
+set -x # echo all commands used for debugging purposes
+
+DOWNLOAD=download.eclipse.org/lsp4j/$LSP4J_PUBLISH_LOCATION
+DOWNLOAD_MOUNT=/home/data/httpd/$DOWNLOAD
+ARTIFACTS=https://ci.eclipse.org/lsp4j/job/${PROJECT}/$LSP4J_BUILD_NUMBER/artifact
+
+if [ "$PROJECT" == "lsp4j" ]; then
+    ARTIFACTS_REPO_TARGET=$ARTIFACTS/build
+else
+    echo "Missing repo location of $PROJECT"
+    exit 1
+fi
+
+echo Using download location root of "https://$DOWNLOAD"
+echo Using artifacts location root of $ARTIFACTS
+
+echo Testing to make sure we are publishing to a new directory
+test ! -e $DOWNLOAD_MOUNT
+
+echo Testing to make sure artifacts location is sane
+wget -q --output-document=/dev/null $ARTIFACTS_REPO_TARGET
+
+ECHO=echo
+if [ "$DRY_RUN" == "false" ]; then
+    ECHO=""
+else
+    echo Dry run of build:
+fi
+
+$ECHO mkdir -p $DOWNLOAD_MOUNT
```

Comment:
this wont do a drops deployment correct?
seems like it was not done for some releases.
am not sure if this is nice for people who want to download the repo
https://download.eclipse.org/lsp4j/downloads/drops/?d

Reply:
The "drops" is a separate issue and if someone needs it they can request and I will look into doing that. Do you need it @cdietrich?

Reply:
not right now